mlx_audio/server.py

```diff
@@ -390,6 +487,89 @@ def stop_audio():
         )
 
 
+@app.get("/languages")
```

Comment:
This looks good!
However, the approach will change. Server won't have languages or models endpoints to simplify it because there are model specific.

We can return directly from the model config or object.

See these:
- https://github.com/Blaizzy/mlx-audio/pull/153
- https://github.com/Blaizzy/mlx-audio/pull/154